### PR TITLE
Update apis and added insert facility

### PR DESCRIPTION
**Pull request by manjilthapa**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import "./App.css";
+import * as facilityService from "../src/services/FacilityService";
 
 function App() {
+  const data = facilityService.getAllFacilities();
```
Comment (mfateminia) [inline src/App.tsx:6]: what is this?

File: src/services/FacilityService.ts [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
-      localStorage.setItem(KEYS.facilities, JSON.stringify([]));
-    return myPromise.then((value) => value);
-  }
-  
-  const myPromise = new Promise((resolve, reject) => {
+  facilities: "facilities",
+};
+export const getAllFacilities = async (): Promise<Facility[]> => {
+  const facilities: Facility[] = JSON.parse(
+    localStorage.getItem(KEYS.facilities) || "[]"
```
Comment (mfateminia) [inline src/services/FacilityService.ts:8]: if you are already setting this to empty array, why do u need to return empty array again at the end of the function?

File: src/App.tsx (hunk 1 above)
Comment (manjilthapa) [reply src/App.tsx:6]: this was just for testing. It will be remove later.

File: src/services/FacilityService.ts (hunk 2 above)
Comment (manjilthapa) [reply src/services/FacilityService.ts:8]: Right, Forget to remove that. I will change and do PR again. Thank youe

**Review by mfateminia — APPROVED** [review]

**Pull request merged**